compiler/resolution/resolveFunction.cpp

```diff
@@ -487,11 +487,14 @@ void resolveSpecifiedReturnType(FnSymbol* fn) {
       fn->retType = retType;
     }
 
-    if (fn->isIterator() == true && fn->iteratorInfo == NULL) {
+    if (fn->isIterator() == true && fn->iteratorInfo == NULL &&
```

Comment:
As long as you're touching this line anyway could you please remove the unnecessary  `== true`?